### ISSUE TITLE
z-coordinate value not proportional with x and y.

### ISSUE THREAD

**Issue by philiprkk** [google/mediapipe#1988]
When visualizing landmark positions in full-body tracking, the z value seems to be upscaled dramatically.
I've attached a screenshot of the landmark coordinates represented in 3D space. I'm I doing something wrong or is the data supposed to look like this?
![117107754-63ffe300-ad82-11eb-9620-06af75e07641](https://user-images.githubusercontent.com/57251565/117256594-2adf7580-ae4b-11eb-904b-d9cbeddb5886.png)

Also, It seems when standing completely straight it also estimates the feet being well behind the hip and knees. This seems consistent in multiple scenarios. 
![image (2)](https://user-images.githubusercontent.com/57251565/117256637-3763ce00-ae4b-11eb-868f-9b8f1fd25a8b.png)

Is this behavior expected?

**Comment by aoxipo** [comment]
The X axis and the Z axis share the same scale, Maybe you can get inspiration from it.

**Comment by sn1c3** [comment]
@philiprkk Did you find a solution?

@aoxipo  Is there a recommended way to scale the z axis? 

So x and y refer to the width and height of the image. Z does only contain values somewhere between -1 <= 0 <= 1 (depending on the image). To make the 3d visualization look realistic, I have to scale the z axis with a factor, in the best case something that correlates to the distance from the camera.


This is the visualization of a picture: (original z axes)
<img src="https://user-images.githubusercontent.com/12408751/121202787-dc702d00-c875-11eb-8a5d-35db237982bd.png" alt="drawing" width="500"/>

The original depth would more look like this: (I set the z-axes to 20)
<img src="https://user-images.githubusercontent.com/12408751/121205087-aa5fca80-c877-11eb-9887-d16122a0ca38.png" alt="drawing" width="500"/>

**Comment by philiprkk** [comment]
@sn1c3 I haven't found a solution either other than scaling down the z-coordinate to something that seems realistic, like what you have done. The aspect ratio of the actual webcam texture used before applying the mediapipe solution to it seems to affect it.

**Comment by sn1c3** [comment]
@philiprkk They published a pose_world_landmarks for a better transposition with mediapipe0.8.6

**Comment by brndusic** [comment]
@sn1c3 at the end have you found any solution for this. What scale have you used?

**Comment by sn1c3** [comment]
hi @brndusic, 
take a look at [pose_world_landmarks](https://google.github.io/mediapipe/solutions/pose.html#pose_world_landmarks) from version v0.8.6. With pose_world_landmarks, they return normalized landmarks.

**Comment by brndusic** [comment]
Hi @sn1c3, I have tried with latest version v0.8.8.1 and with one you mentioned and always had the same problem.

Here are some of mine Z results from pose_world_landmarks:

```
part        left_z_index    right_z_index
Foot index  0.2015          0.1068
Heel        0.2444          0.1682
Ankle       0.2197          0.1856
Knee        0.0287          -0.0305
Hip         0.0326          -0.0306
Shoulder    -0.0918         -0.054
Elbow       -0.0779         -0.1037
Wrist       -0.0549         -0.1751
Ear         -0.2043         -0.201
Eye         -0.2994         -0.3097
```

As you can see my person is bent forward, and this is the case with all my images.

![image](https://user-images.githubusercontent.com/2099985/141697356-88aae5a2-1260-4506-828f-22001252d08a.png)

**Comment by sn1c3** [comment]
Hi @brndusic, 
you are right. I think my only issue was with the proportions of x,y and z. That got solved by the pose_world_landmarks. 

Nevertheless, in my case I need useful depth information during video streams. For precise movements, mediapipe is still too inaccurate. I solved my problem with a Realsense L515 camera, which returns pretty good depth information. So right now, I am using pose_world_landmarks for x and y and for the depth I use a smoothened depth value from a Realsense cam
